src/QMCDrivers/QMCCostFunction.cpp

```diff
@@ -290,7 +290,19 @@ void QMCCostFunction::checkConfigurations()
         //allocate vector
         std::vector<Return_t> Dsaved(NumOptimizables, 0.0);
         std::vector<Return_t> HDsaved(NumOptimizables, 0.0);
+
+        std::vector<Return_ct> cDsaved(NumOptimizables, 0.0);
```

Comment:
As I explained above Return_ct name is bad and cDsaved is bad as well.

Reply:
Fixed.